src/controller/src/clusters.rs

```diff
@@ -124,6 +179,15 @@ impl ReplicaLocation {
             }
         }
     }
+
+    pub fn billed_as(&self) -> Option<&str> {
+        match self {
+            ReplicaLocation::Managed(ManagedReplicaLocation { billed_as, .. }) => {
+                billed_as.as_deref()
+            }
+            _ => None,
```

Comment:
Does this mean that unmanaged clusters cannot ever have billed as set?

Reply:
`ReplicaLocation::Managed` is unrelated to the concept of managed vs. unmanaged clusters. I believe that `ReplicaLocation::Unmanaged` is only used for tests, and that in production this will always be `ReplicaLocation::Managed`. We should definitely rename those.